src/styles/css/main.stylesheet.css

```diff
@@ -500,6 +501,44 @@ input[type='text']:focus {
     position: relative;
 }
 
+/* expansion panel styles - start */
+.v-expansion-panel {
```

Comment:
we shouldn't overwrite library components styles globally, lets's move this to the components style tag, and wrap it inside a custom class?

Reply:
I will fix it next while refactoring the CSS for this component.